third_party/move/move-core/types/src/language_storage.rs

```diff
@@ -280,27 +254,29 @@ pub struct FunctionTag {
     pub abilities: AbilitySet,
 }
 
-/// Represents the initial key into global storage where we first index by the address, and then
-/// the struct tag
-#[derive(Serialize, Deserialize, Debug, PartialEq, Hash, Eq, Clone, PartialOrd, Ord)]
-pub struct ResourceKey {
-    pub address: AccountAddress,
-    pub type_: StructTag,
-}
-
-impl ResourceKey {
-    pub fn address(&self) -> AccountAddress {
-        self.address
-    }
-
-    pub fn type_(&self) -> &StructTag {
-        &self.type_
-    }
-}
-
-impl ResourceKey {
-    pub fn new(address: AccountAddress, type_: StructTag) -> Self {
-        ResourceKey { address, type_ }
+impl FunctionTag {
+    /// Returns a canonical string representation of the function tag.
+    pub fn to_canonical_string(&self) -> String {
+        let fmt_list = |l: &[TypeTag]| -> String {
+            l.iter()
+                .map(|t| t.to_canonical_string())
+                .collect::<Vec<_>>()
+                .join(", ")
+        };
+        // Note that we put returns in parentheses. This ensures that when functions used as type
```

Comment:
Great. I think you didn't mention this original fix in the PR description. ;-)

Reply:
Forgot to update 👍 done